Functional programming/Problem 1/partial_application.js

```diff
@@ -0,0 +1,15 @@
+function F() {
```

Comment:
Using more intuitive names will increase your code [readability](http://javarevisited.blogspot.com/2014/10/10-java-best-practices-to-name-variables-methods-classes-packages.html).